src/main/java/uk/ac/ebi/atlas/hcalandingpage/HcaHumanExperimentService.java

```diff
@@ -18,11 +20,16 @@ public HcaHumanExperimentService(ExperimentTrader experimentTrader,
         this.hcaHumanExperimentDao = hcaHumanExperimentDao;
     }
 
-    public ImmutableSet<Experiment> getPublicHumanExperiments(String characteristicName, String characteristicValue) {
-        return hcaHumanExperimentDao.fetchExperimentAccessions(characteristicName, characteristicValue)
+    public ImmutableSet<Experiment> getPublicHumanExperiments(String characteristicName,
+                                                              Set<String> characteristicValues) {
+
+        Set<String> experimentAccessions = hcaHumanExperimentDao.fetchExperimentAccessions(characteristicName,
+                                                                                            characteristicValues);
+        return experimentAccessions
                 .stream()
                 .map(experimentTrader::getPublicExperiment)
                 .filter(experiment -> experiment.getSpecies().isUs())
                 .collect(toImmutableSet());
     }
+
```

Comment:
Remove blank space.

Reply:
The tool is giving this extra black space before and end of class. 
I am not sure how the tool is working in you're development IDE.
Please, let me know to fix it if required.